scripts/get_users_meta_docs.js

```diff
@@ -0,0 +1,102 @@
+const inquirer = require('inquirer'),
+      PouchDB = require('pouchdb-core'),
+      fs = require('fs'),
+      path = require('path'),
+      minimist = require('minimist');
+
+PouchDB.plugin(require('pouchdb-adapter-http'));
+
+const argv = minimist(process.argv.slice(2));
+
+if(argv['h'] || argv['help']) {
+  console.log(`Display or save telemetry and feedback docs.
+
+Usage:
+    ${process.argv[1]} -h | --help
+    ${process.argv[1]} --mode (interactive | batch) --type (telemetry | feedback) [medic users meta db URL]
+
+Options:
+    -h --help     Show this screen.
+    --mode        Interactive or Batch mode (Output docs to stdout)
+    --type        Meta document type to fetch. 
+
+Example:
+    ${process.argv[1]} --interactive --type telemetry http://admin:pass@localhost:5984/medic-users-meta
+`);
+  process.exit(0);
+}
+
+const mode = argv['mode'];
+const type = argv['type'];
+const couchUrl = argv['_'][0] || 'http://admin:pass@localhost:5984/medic-users-meta';
+
+const db = PouchDB(couchUrl);
+
+let docIndex = 0;
+let docs;
+
+const actionChoices = [
+  { name: 'Display next', value: 'next' }, 
+  { name: 'Display previous', value: 'previous' }, 
+  { name: 'Save current in working directory', value: 'save_current' }, 
+  { name: 'Save all in working directory', value: 'save_all' },
+  { name: 'Quit', value: 'quit' }
+];
+
+const actionQuestions = [{
+  type: 'list',
+  name: 'action',
+  message: 'Do you want to?',
+  choices: actionChoices
+}];
+
+const askActionQuestions = () => {
+  inquirer.prompt(actionQuestions)
```

Comment:
Super nice to see folks starting to use async/await. Consider using async here also instead of mixing styles.